scripts/release/test/rpm/run_centos.sh

```diff
@@ -9,6 +9,11 @@ echo
 date "+build_release begin TEST stage %Y%m%d_%H%M%S"
 echo
 
+if [ "$CHANNEL" = beta ]; then
+    echo "There is currently no support for RPM beta packages. Exiting RPM test stage..."
+    exit 0
```

Comment:
should exit code be 1?

Reply:
No, because it should just exit out of the script and not fail the build.